standup/auth0/views.py

```diff
@@ -59,7 +58,7 @@ def get(self, request):
                     'Unable to authenticate with Auth0 at this time. Please refresh to '
                     'try again.'
                 )
-                return HttpResponseRedirect(reverse('users.loginform'))
+                return HttpResponseRedirect(reverse(settings.AUTH0_SIGNIN_VIEW))
```

Comment:
There are a few changes like this one which generalize this code so we can librarificate it more easily later on.